### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
fixed bugs

## PR Checklist

- [x] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [ ] If there is an issue related with this PR, point it out here.

## What is the new state of the documentation article?

Fixes/Implements/Closes #[Issue Number].

### CODE REVIEW

**Review by etabakov — CHANGES_REQUESTED** [review]
As the preferred spelling is American English, can you revert to "labeled" so we can accept the other changes. Thanks.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Help us keep a healthy and open community. We expect all participants in this pr
 
 1. Always update to the most recent master release; the bug may already be resolved.
 2. Search for similar issues in the issues list for this repo; it may already be an identified problem.
-3. If this is a bug or problem that is clear, simple, and is unlikely to require any discussion -- it is OK to open an issue on GitHub with a reproduction of the bug and screenshots. If possible, submit a Pull Request and fix the bug yourself (jump down to the [Submitting a PR](#submitting-a-pr) section).
+3. If this is a bug or problem that is clear, simple and is unlikely to require any discussion -- it is OK to open an issue on GitHub with a reproduction of the bug and screenshots. If possible, submit a Pull Request and fix the bug yourself (jump down to the [Submitting a PR](#submitting-a-pr) section).
```
Comment (etabakov) [inline CONTRIBUTING.md:25]: I think the previous state was the correct usage of the Oxford comma rule.

Comment (sakshamtaneja21) [reply CONTRIBUTING.md:25]: ok i got it ill add some changes and tell you

**Review by etabakov — APPROVED** [review]

**Comment by etabakov** [comment]
Hey, thanks for your contribution! We have a special program for our first-time contributors and you are eligible to participate in it. If you haven't applied already, all you need to do to claim your prize is to fill our [Application Form](https://goo.gl/forms/lPKpXzG4F1EhfKM72).

Regards from the whole team 🥇

**Pull request merged**